packages/components/src/ui/styles/theme/config.js

```diff
@@ -0,0 +1,370 @@
+	cardBorderRadius: '2px',
+	cardPaddingX: space( 3 ),
+	cardPaddingY: space( 3 ),
+	cardPadding: flow( get( 'cardPaddingX' ), get( 'cardPaddingY' ) ),
+	cardHeaderFooterPaddingY: space( 1 ),
+	cardHeaderHeight: '44px',
+};
+
+const CHECKBOX_PROPS = {
+	checkboxBoxShadow: 'none',
+	checkboxSize: '16px',
+};
+
+const FLEX_PROPS = {
+	flexGap: space( 2 ),
+	flexItemMarginRight: get( 'flexGap' ),
+};
+
+const LINK_PROPS = {
+	linkColor: get( 'colorAdmin' ),
+	linkColorHover: get( 'colorAdmin' ),
+	linkColorActive: get( 'colorAdmin' ),
+	linkColorFocus: get( 'colorAdmin' ),
+};
+
+const MENU_PROPS = {
+	menuItemBorderWidth: '1px',
+	menuItemFocusBackgroundColor: 'transparent',
+	menuItemFocusBorderColor: get( 'colorAdmin' ),
+	menuItemFocusTextColor: get( 'menuItemFocusBorderColor' ),
+	menuItemFocusBoxShadow: get( 'controlBorderSubtleColor' ),
+	menuItemActiveBackgroundColor: get( 'controlBackgroundColor' ),
+	menuItemActiveBorderColor: get( 'menuItemFocusBorderColor' ),
+	menuItemActiveTextColor: get( 'colorText' ),
+	menuItemActiveBoxShadow: get( 'controlBorderSubtleColor' ),
+	menuItemHeight: '30px',
+	menuItemHeightLarge: `calc(${ get( 'menuItemHeight' ) } * 1.2)`,
+	menuItemHeightSmall: `calc(${ get( 'menuItemHeight' ) } * 0.8)`,
+	menuItemHeightXLarge: `calc(${ get( 'menuItemHeight' ) } * 1.4)`,
+	menuItemHeightXSmall: `calc(${ get( 'menuItemHeight' ) } * 0.67)`,
+	menuItemHeightXXSmall: `calc(${ get( 'menuItemHeight' ) } * 0.4)`,
+};
+
+const PANEL_PROPS = {
+	panelHeaderPadding: `${ space( 3 ) } ${ space( 4 ) }`,
+	panelBodyPadding: `${ space( 2 ) } ${ space( 4 ) } ${ space( 3 ) }`,
+};
+
+const RADIO_PROPS = {
+	radioBoxShadow: get( 'checkboxBoxShadow' ),
+	radioSize: get( 'checkboxSize' ),
+	radioDotSize: '10px',
+};
+
+const SEGMENTED_CONTROL_PROPS = {
+	segmentedControlFontSize: '12px',
+	segmentedControlBackgroundColor: get( 'controlBackgroundColor' ),
+	segmentedControlBorderColor: get( 'controlBorderColor' ),
+	segmentedControlBackdropBackgroundColor: get( 'controlSurfaceColor' ),
+	segmentedControlBackdropBorderColor: get( 'controlBorderColor' ),
+	segmentedControlBackdropBoxShadow: 'transparent',
+	segmentedControlButtonColorActive: get( 'controlBackgroundColor' ),
+};
+
+const SLIDER_PROPS = {
+	sliderThumbBorderColor: 'transparent',
+	sliderThumbBoxShadow: 'none',
+	sliderThumbBoxShadowSizeFocus: '3px',
+	sliderThumbBoxShadowColorFocus: get( 'colorAdminRgba20' ),
+	sliderThumbBackgroundColor: get( 'colorAdmin' ),
+};
+
+const SWITCH_PROPS = {
+	switchBackdropBackgroundColor: get( 'lightGray900' ),
+	switchBackdropBackgroundColorActive: get( 'colorAdmin' ),
+	switchBackdropBorderColor: get( 'lightGray900' ),
+	switchBackdropBorderColorActive: get( 'colorAdmin' ),
+	switchBackdropBorderColorFocus: get( 'white' ),
+	switchToggleBackgroundColor: get( 'colorTextInverted' ),
+	switchToggleBackgroundColorActive: get( 'colorTextInverted' ),
+	switchToggleBoxShadow: 'none',
+	switchPaddingOffset: '6px',
+};
+
+const BASE_THEME = {
```

Comment:
Is this what constitutes a theme?

Reply:
Yup, just one big JS object that get's passed to `createStyleSystem`.

Reply:
I'm fine with this being so big as long as it's not generating CSS variables though I'd consider removing all these thimble options and starting small personally. Do we know for sure that we want to tweak everything in themes?

Reply:
I think the best strategy would be to basically remove all the colors except for the obvious theme variables that align with the ones in `base-styles` and the `admin-scheme` mixin.

Then we can add more as we go along when we feel it's necessary for them to exist.

I'm not sure what to do about `--wp--preset--` variables from global styles. It seems like they should be accounted for somehow in this whole system but I'll need to think more about how to actually make that happen.

Reply:
I think the presets variable are completely independent of the UI components, they are important for themes and blocks output but not for UI components.